puppet/modules/chassis/manifests/site.pp

```diff
@@ -15,7 +15,7 @@
 	$sitename       = 'Chassis Site',
 ) {
 	$extra_hosts = join($hosts, ' ')
-	$server_name = rstrip("${name} ${extra_hosts}")
+	$server_name = rstrip($name)
```

Comment:
```suggestion
	$server_name = rstrip(${name})
```

Should this remain in curly brackets?

Also what about the hosts array of hosts, is this affected by this change?

Also, what about these instances?
https://github.com/Chassis/Chassis/blob/5f8efbae510a931365f5294bcddca2ea7a4715db/puppet/modules/chassis/manifests/network.pp#L19

https://github.com/Chassis/Chassis/blob/5f8efbae510a931365f5294bcddca2ea7a4715db/puppet/modules/chassis/manifests/network.pp#L22

Reply:
>Should this remain in curly brackets?
Negative.

>Also what about the hosts array of hosts, is this affected by this change?
There should only be more than one host if they are using multisite.

>Also, what about these instances?
They are fine as they are for multisites.